explainer-server/app/views/explainEditor.scala.html

```diff
@@ -68,6 +66,24 @@
             }
         }
 
+        /*
+         * This function (and the accompanying handler) exists because there doesn't seem to exist a way
+         * to generate a checkbox using ScalaTags (ScalaJS's HTML generation library)
+         */
+        function installDisplayTypeTickBox(displayType){
+            console.log(displayType);
+            var checkedstring = displayType=='Expandable' ? ' checked ' : ''
+            $(".explainer-editor__displayType-inner").html('<input class="explainer-editor__displayType-checkbox" type="checkbox" name="displayType" value="long" '+checkedstring+'> Long explainer')
+        }
+        $(document).delegate(".explainer-editor__displayType-checkbox", 'click', function() {

```

Comment:
`delegate` allows setting an event on a DOM element that has not been rendered yet (reason why I used it), but I can easily change `click` to `change`. Now it is 

```
        $(document).delegate(".explainer-editor__displayType-checkbox",'change',function() {
            var $this = $(this);
            if ($this.is(':checked')) {
                ExplainEditorJS().setDisplayType("@id","Expandable")
            } else {
                ExplainEditorJS().setDisplayType("@id","Flat")
            }
        });
```
works as expected